src/gui_common/SpeciesDetailsPanel.tscn

```diff
@@ -0,0 +1,101 @@
+[gd_scene load_steps=9 format=2]
+
+[ext_resource path="res://src/gui_common/thrive_theme.tres" type="Theme" id=1]
+[ext_resource path="res://src/gui_common/SpeciesDetailsPanel.cs" type="Script" id=2]
+[ext_resource path="res://src/gui_common/fonts/Lato-Bold-AlmostSmaller.tres" type="DynamicFont" id=3]
+[ext_resource path="res://src/gui_common/PhotographablePreview.tscn" type="PackedScene" id=4]
+[ext_resource path="res://src/gui_common/CustomRichTextLabel.tscn" type="PackedScene" id=5]
+[ext_resource path="res://src/gui_common/SpeciesPreview.cs" type="Script" id=6]
+[ext_resource path="res://src/microbe_stage/CellHexesPreview.cs" type="Script" id=7]
+[ext_resource path="res://src/thriveopedia/fossilisation/FossilisationDialog.tscn" type="PackedScene" id=8]
+
+[node name="SpeciesDetailsPanel" type="PanelContainer"]
```

Comment:
I think this should be a `Control` so that it can be added in any Container. The current design will result in fixed margin and so on. It may create GUI designing issues.